docs/architecture.md

```diff
@@ -0,0 +1,18 @@
+# Architecture
+
+This project is divided in 4 main parts:
+
+- `src/frontend`: A frontend library, published as `@jitsi-magnify/frontend-lib` on NPM. 
+- `src/demo/frontend`: A frontend demo, published as `@jitsi-magnify/frontend-demo`: a simple react app that show a typical usecase for the components, the views, the code provided by the front library
+- `src/magnify`: A backend library, published as `magnify` on PyPI, that exposes a typical django configuration, and some apps (currently only one)
+- `src/demo/backend`: A backend demo: a django project that show how to integrate the apps in an actual project
+
+The demo projects are designed for the usecase of openfun, but can be used a a basis for your own project. The apps published on PyPI can be used independantly in your own django project.
```

Comment:
Yes, of course, but nevertheless, it will somehow be tailored to FUN use-case in the choices we will have to do. The libraries would be much more generic. You may use the demo out of the box with some settings, but to create a fully personalized experience, you might need to create it from the library (that will contain most of the logic)